Corona-Warn-App/src/main/res/layout/contact_diary_location_list_line.xml

```diff
@@ -7,7 +7,7 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:minHeight="60dp"
-        android:background="@color/colorSurface2">
+        style="@style/cardGrey">
```

Comment:
use contactDiaryCard instead of cardGrey since it will not match the desired Colors in dark mode

Reply:
ok, I have to check, because I added the ripple effect here

Reply:
I moved the ripple effect to contactDiaryCard